### PR TITLE
Repository Permission for Flock Plugin

### PR DESCRIPTION
**Pull request by ajaymann**
# Description

Github Repository - https://github.com/jenkinsci/flock-plugin
Hosting Request - https://issues.jenkins-ci.org/browse/HOSTING-816
Maintainers - @ajaymann

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

#### Merge permission to GitHub repository
- [x] Check this if newly added person also needs to be given merge permission to the GitHub repo.

### CODE REVIEW

**Comment by yatinsns** [comment]
LGTM

**Comment by ajaymann** [comment]
We are good to go, what are the next steps to get this merged?

File: permissions/plugin-flock.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "flock"
+github: "jenkinsci/flock-plugin"
+paths:
+- "org/jenkins-ci/plugins/flock"
```
Comment (slide) [inline permissions/plugin-flock.yml]: Your path needs to use the groupId for your project, not the parent pom. 

https://github.com/jenkinsci/flock-plugin/blob/master/pom.xml#L12

So, your path would be io/jenkins/plugins/flock

Comment (ajaymann) [reply permissions/plugin-flock.yml]: Thank you for providing the feedback on this, I've pushed again with the changes.

**Pull request merged**